build.gradle

```diff
@@ -127,6 +128,9 @@ apply from: file('gradle/ide/intellij-idea.gradle')
 apply from: file('gradle/ide/eclipse.gradle')
 
 // Validation tasks
+apply from: file('gradle/hacks/findbugs.gradle')
```

Comment:
Hmm! Locally this made precommit happy

But https://github.com/apache/lucene-solr/pull/1816/checks?check_run_id=1069510797 still is running into the same error 🤔

Reply:
I think the problem may be in clashing JARs - errorprone annotations are imported in a different place of the build (Erick worked on this), without any static analysis. I can take a look but later.